package/CHANGELOG

```diff
@@ -17,11 +17,12 @@ The rules for this file:
          lilyminium, daveminh, jbarnoud, yuxuanzhuang, VOD555, ianmkenney,
          calcraven，xiki-tempula, mieczyslaw, manuel.nuno.melo, PicoCentauri,
          hanatok, rmeli, aditya-kamath, tirkarthi, LeonardoBarneschi, hejamu,
-         biogen98
+         biogen98, orioncohen
 
   * 2.0.0
 
 Fixes
+  * Fixed 'sphzone' behavior to be consistent with 'around' (Issue #2915)
```

Comment:
Could you now update this CHANGELOG message to be more informative?

Reply:
Done! Thanks for the help Lily! (loved your PyCon talk btw)

Reply:
Thank you @oriencohen! :-)